packages/utils/telemetry-utils/src/test/tsconfig.cjs.json

```diff
@@ -0,0 +1,17 @@
+{
+	"extends": "@fluidframework/build-common/ts-common-config.json",
+	"compilerOptions": {
+		"rootDir": "./",
+		"outDir": "../../dist/test",
+		"types": ["mocha", "node"],
+		"declaration": false,
+		"declarationMap": false,
+		"skipLibCheck": true,
+	},
+	"include": ["./**/*"],
+	"references": [
+		{
+			"path": "../..",
```

Comment:
This reference is not correct. It is referencing the ESM build of this package rather than the CJS version.
Another case where we silently build and test the wrong type. This "CJS" build of test is actually building ESM, which is why the bad reference doesn't trigger any build failures. :(
It looks like the head of telemetry-utils corrects this. I think it is two commits ahead of what was merged to main in this PR.